lib/scala/profiling-utils/src/test/java/org/enso/profiling/snapshot/HeapDumpGeneratorTest.java

```diff
@@ -0,0 +1,25 @@
+package org.enso.profiling.snapshot;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HeapDumpGeneratorTest {
+
+  @Test
+  public void generateHeapDump() throws IOException {
+    String heapDumpFileName =
+        getClass().getSimpleName() + "-" + System.currentTimeMillis() + ".hprof";
```

Comment:
Might be easier to locate if this has a human-readable time format.

Reply:
It is supposed to be removed after the test anyway. The actual heap dump file created by the language server will be in a human-readable format.